src/components/TextInput/index.native.tsx

```diff
@@ -1,13 +1,19 @@
-import React, {forwardRef, useEffect} from 'react';
-import {AppState, Keyboard} from 'react-native';
+import React, {Component, ForwardedRef, forwardRef, useEffect} from 'react';
+import {AppState, Keyboard, TextInputProps} from 'react-native';
+import {AnimatedProps} from 'react-native-reanimated';
 import useThemeStyles from '@styles/useThemeStyles';
 import BaseTextInput from './BaseTextInput';
-import * as baseTextInputPropTypes from './BaseTextInput/baseTextInputPropTypes';
+import BaseTextInputProps from './BaseTextInput/types';
 
-const TextInput = forwardRef((props, ref) => {
+// eslint-disable-next-line react/function-component-definition
```

Comment:
Maybe it makes sense to update it to function component definition and get rid of the eslint ignore